### ISSUE TITLE
Item-info unrecognized mods 

### ISSUE THREAD

**Issue by StochasticBoris** [Lailloken/Lailloken-UI#431]
On certain crafts, for example crafting flasks with alteration orbs, the item-info module doesn't work correctly. It either displays a tooltip with no mods, or shows the message "there's nothing to display". 

This only happens about 50% of the time, at times it works as expected. This issue is not specific to any newly added mods, old flask mods that it used to recognize aren't working.

Tested on 1.54.4 and 1.54.5.

**Comment by Lailloken** [comment]
I tried to replicate this but I can't _right now_. Your issue was probably due to high server load, i.e. the item took longer than usual to generate, and the tool tried to copy the item's info before it was ready. Could you try it again when it's morning in your timezone?

I may have to increase the delay to account for high server load during league-start.

**Comment by StochasticBoris** [comment]
Thank you for looking into it. I did a fresh reinstall and it seems to have fixed my problems.

I have been updating the program by overwriting its folder contents with new patch zip files (wanted to preserve previous settings), which may have introduced some corruptions over time.
I now understand that it is possible to update from within the program by selecting a minor version, then the "apply and restart" button appears. Perhaps consider always showing the "apply update" button, but greyed out, so that less bright users like me don't get confused about the process.

Closing the issue since the error was due to my fault.
